poollet/volumepoollet/vcm/generic.go

```diff
@@ -63,6 +102,8 @@ func (g *Generic) relist(ctx context.Context, log logr.Logger) error {
 	g.mu.Lock()
 	defer g.mu.Unlock()
 
+	g.notify(res.VolumeClasses)
```

Comment:
We should somehow merge `notify` with the process afterwards (getting capabilities etc.) - we are iterating too often on the same values.